Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Математические_формулы.Методы
+{
+    class Program
+    {
+        static void Main(string[] args)
+  
+        {
+            Console.WriteLine("Введите произвольную переменную");
+            var x = double.Parse(Console.ReadLine());
+            printf(x);
+            
+            Console.ReadKey();
+
+
+        }
+
+        private static double printf(double x)

```

Comment:
Неудачное название метода. Во-первых, метод ничего не печатает (почему print?). Во-вторых, в C# методы принято называть с большой буквы. Например,
```suggestion
        private static double F(double x)
```
или
```suggestion
        private static double GetFormulaValue(double x)

```